gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/EmailServiceImpl.java

```diff
@@ -247,6 +248,10 @@ private Map<String, Object> prepareEmailParams(Domain domain, Application client
             claims.put(Claims.AUD, user.getClient());
         }
 
+        if (template == io.gravitee.am.model.Template.REGISTRATION_CONFIRMATION) {
```

Comment:
do we need this? 

In management we have something like this
     getTokenPurpose(template)
                .ifPresent(purpose -> claims.put(ConstantKeys.CLAIM_TOKEN_PURPOSE, purpose));